src/fields/BaseRelationField.php

```diff
@@ -489,6 +506,27 @@ public function normalizeValue(mixed $value, ?ElementInterface $element = null):
                     Craft::configure($query, $source['criteria']);
                 }
             }
+
+            if ($this->relateAncestors || $this->branchLimit) {
```

Comment:
at the time I didn't know if you can set branch limit and relateAncestors separately. Since we're only showing branch limit if relateAncestors is true then yea i'll just check for relateAncestors here.

Reply:
👍